pwnlib/fmtstr.py

```diff
@@ -91,8 +91,12 @@ def send_payload(payload):
 """
 from __future__ import division
 
+import copy
+import functools
```

Comment:
I could not find any other occurrence of imported `copy` and `functools`. Are they actually used?

Reply:
Right, removed